.github/workflows/test-tpg.yml

```
@@ -42,21 +40,22 @@ jobs:
       with:
         repository: ${{ github.event.inputs.owner }}/${{ github.event.inputs.repo }}
         ref: ${{ github.event.inputs.branch }}
-        path: provider
+        path: repo
         fetch-depth: 2
     - name: Cache Go modules and build cache
       uses: actions/cache@v3
       with:
         path: |
           ~/go/pkg/mod
           ~/.cache/go-build
-        key: ${{ runner.os }}-test-${{ github.event.inputs.repo }}-${{ hashFiles('**/go.sum') }}
+        key: ${{ runner.os }}-test-${{ github.event.inputs.repo }}-${{ hashFiles('repo/go.sum', 'repo/google-beta/transport/**', 'repo/google-beta/tpgresource/**', 'repo/google-beta/acctest/**', 'repo/google-beta/envvar/**', 'repo/google-beta/sweeper/**', 'repo/google-beta/verify/**') }}
```

In that case, could this work?
```suggestion
        key: ${{ runner.os }}-test-${{ github.event.inputs.repo }}-
          ${{
            hashFiles(
              'repo/go.sum',
              'repo/google-beta/transport/**',
              'repo/google-beta/tpgresource/**',
              'repo/google-beta/acctest/**',
              'repo/google-beta/envvar/**',
              'repo/google-beta/sweeper/**',
              'repo/google-beta/verify/**')
          }}
```

---

That might work ! Lets try it !